proto/zetachain/zetacore/observer/query.proto

```diff
@@ -150,6 +151,30 @@ service Query {
       returns (QueryAllChainNoncesResponse) {
     option (google.api.http).get = "/zeta-chain/observer/chainNonces";
   }
+  // Queries the TssFundMigratorInfo for a specific chain
+  rpc TssFundsMigratorInfo(QueryTssFundsMigratorInfoRequest)
+      returns (QueryTssFundsMigratorInfoResponse) {
+    option (google.api.http).get = "/zeta-chain/observer/get_tss_fund_migrator";
+  }
+
+  // Queries all TssFundMigratorInfo
+  rpc TssFundsMigratorInfoAll(QueryTssFundsMigratorInfoAllRequest)
+      returns (QueryTssFundsMigratorInfoAllResponse) {
+    option (google.api.http).get =
+        "/zeta-chain/observer/get_all_tss_fund_migrators";
+  }
+}
+
+message QueryTssFundsMigratorInfoAllRequest {}
+
+message QueryTssFundsMigratorInfoAllResponse {
+  repeated TssFundMigratorInfo tss_funds_migrators = 1
+      [ (gogoproto.nullable) = false ];
+}
+
+message QueryTssFundsMigratorInfoRequest { int64 chain_id = 1; }
+message QueryTssFundsMigratorInfoResponse {
+  TssFundMigratorInfo tss_funds_migrator = 1 [ (gogoproto.nullable) = false ];
```

Comment:
We should stick to camel case for now if other queries are using camel case
I may also think there is a problem with TypeScript client generation when using snake case (this is why originally Ignite templates was generating with camel case)

Reply:
3ce96875387aecd2302b74b2d6f9d2bce8e30e26